From 07e597f72f32f2e2b14b16db4f1f0d2d52ced28f Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Tue, 28 Oct 2014 18:45:19 +0100
Subject: [PATCH 16/24] profile-resetter: do not tick send-settings by default

When the user wishes to reset his preferences, Chromium would
suggest to send over the old settings. GTFO.
---
 chrome/browser/resources/options/reset_profile_settings_overlay.html | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/resources/options/reset_profile_settings_overlay.html b/chrome/browser/resources/options/reset_profile_settings_overlay.html
index 98d5248..66bef84 100644
--- a/chrome/browser/resources/options/reset_profile_settings_overlay.html
+++ b/chrome/browser/resources/options/reset_profile_settings_overlay.html
@@ -23,7 +23,7 @@
   </div>
   <div id="feedback-bar" class="gray-bottom-bar checkbox">
     <span class="controlled-setting-with-label">
-      <input id="send-settings" type="checkbox" checked>
+      <input id="send-settings" type="checkbox">
       <span>
         <label for="send-settings" i18n-content="resetProfileSettingsFeedback">
         </label>
-- 
2.4.0

